src/scss/mixins/_radio-checkbox.scss

```diff
@@ -119,28 +121,47 @@
 /// @param {string} $class - The block level class name (BEM naming convention).
 /// @output Styles to wrap a group of checkboxes or a group of radios for purposes of positioning etc.
 @mixin oFormsGroupContainer($class: 'o-forms') {
-    margin: 0 0 #{-$_o-forms-spacing};
+    margin-top: $_o-forms-checkbox-legend-spacing;
 
-    input + .#{$class}__label {
-        width: 100%; // We do not use display:block to maintain consistent margins (not collapsed).
-        box-sizing: border-box;
-        margin: $_o-forms-checkbox-legend-spacing $_o-forms-checkbox-horizontal-spacing $_o-forms-spacing 0; // margins do not collapse, making the space between checkbox/radio inputs a little larger
-        &:last-of-type {
-            margin-right: 0;
+    // Stacked radios/checkboxes.
+    // @breaking Remove the not selectors by removing the need to override label margins.
+    &:not(#{&}--inline-together):not(#{&}--inline) {
```

Comment:
I struggled with this as I couldn't undo the margin reset with `--inline-together` or `--inline` reliably -- so I instead only apply the margin reset if `--inline-together` or `--inline` are not present. You're right this might cause problems later though, would be great to avoid! I'm open to suggestions though if you can think of a better way without a breaking change.

Reply:
Is it possible to not set default margins, instead only set them on the variants where they're needed? Then you avoid the need to reset them in specific cases.
I'm aware this might be hard to do in practise without a bunch of breaking changes. Just a thought.

Reply:
I'm not sure this is possible without a breaking as we share a class names across groups of checkboxes/radios with a group of styled radios. 
`.o-forms__group {} // stacked checkboxes or radios`
`.o-forms__group .o-forms__group--inline {}  // inline checkboxes or radios`
`.o-forms__group .o-forms__group--inline-together // styled radio buttons`